metadata-ws/src/main/resources/db/changelog/db.changelog-1.0.0.xml

```diff
@@ -0,0 +1,339 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
```

Comment:
how is this xml generated ?

Reply:
Liquibase has a command generateChangeLog to get the changes from the existing DB. The same is used to get changelog from existing master build.

Reply:
so you need to add it to the build phase to automatically generate it